### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by luisincrespo**
Changes proposed in this pull request:

- Add missing `status` value corresponding to successful _publish_ operations.

### CODE REVIEW

**Comment by luisincrespo** [comment]
@googlebot I signed it!

**Comment by dotproto** [comment]
Reaching out to CWS eng now to verify this change.

File: site/en/docs/webstore/webstore_api/items/publish/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ If successful, this method returns a response body with the following structure:
...
+<table><thead><tr><th>Property name</th><th>Value</th><th>Description</th></tr></thead><tbody><tr id="kind"><td><code>kind</code></td><td><code>string</code></td><td>Static string value is always <code>"chromewebstore#item"</code>.</td></tr><tr id="item_id"><td><code>item_id</code></td><td><code>string</code></td><td>The ID of this item.</td></tr><tr id="status"><td><code>status[]</code></td><td><code>list</code></td><td>The status code of this publish operation. It may contain multiple elements from the following list: <code>OK</code>, <code>NOT_AUTHORIZED</code>, <code>INVALID_DEVELOPER</code>, <code>DEVELOPER_NO_OWNERSHIP</code>, <code>DEVELOPER_SUSPENDED</code>, <code>ITEM_NOT_FOUND</code>, <code>ITEM_PENDING_REVIEW</code>, <code>ITEM_TAKEN_DOWN</code>, <code>PUBLISHER_SUSPENDED</code>.</td></tr><tr id="statusDetail"><td><code>statusDetail[]</code></td><td><code>list</code></td><td>Detailed human-comprehensible explanation of the status code above.</td></tr></tbody></table>
```
Comment (CharyCh) [inline site/en/docs/webstore/webstore_api/items/publish/index.md:63]: Do you need to list all possible error code? We actually have 20 different status that could be returned.

**Review by dotproto — APPROVED** [review]
Verified this value with the CWS team. Thanks for the contribution!

**Pull request merged**